rs/replicated_state/src/page_map/storage/tests.rs

```diff
@@ -676,31 +676,33 @@ fn write_overlays_and_verify_with_tempdir(
 /// after every step.
 /// Use unsharded LSMT config.
 fn write_overlays_and_verify_unsharded(instructions: Vec<Instruction>) -> MetricsRegistry {
-    let tempdir = Builder::new()
+    let tdir = Builder::new()
         .prefix("write_overlays_and_verify_unsharded")
         .tempdir()
         .unwrap();
     let metrics =
-        write_overlays_and_verify_with_tempdir(instructions, &lsmt_config_unsharded(), &tempdir);
-    tempdir
-        .close()
```

Comment:
Was the call to `close()` specific to make fuzzing work previously?

Reply:
Yeah, it was the first [attempt](https://github.com/dfinity/ic/commit/be7aa806ec08d1e21f0adf8766ca02d0ac074e54) to enforce the deletion but didn't work. Now, I just opted to do it manually and it works as expected.